solana-programs/anchor/audius-data/lib/lib.ts

```diff
@@ -259,6 +262,39 @@ export const updateAdmin = async ({
   );
 };
 
+/// Verify user with authenticatorKeypair
+type UpdateIsVerifiedParams = {
+  program: Program<AudiusData>;
+  userStgAccount: anchor.web3.PublicKey;
+  verifierKeypair: anchor.web3.Keypair;
+  baseAuthorityAccount: anchor.web3.Keypair;
+  adminKeypair: Keypair;
+  handleBytesArray: number[];
+  bumpSeed: number;
+};
+export const updateIsVerified = async ({
+  program,
+  adminKeypair,
+  userStgAccount,
+  verifierKeypair,
+  baseAuthorityAccount,
+  handleBytesArray,
+  bumpSeed
+}: UpdateIsVerifiedParams) => {
+  return program.rpc.updateIsVerified(
+    baseAuthorityAccount,
```

Comment:
typescript complains here. does this need to be removed?

Reply:
It definitely works though - there's a test for it.
I followed the example in the anchor docs - https://book.anchor-lang.com/chapter_3/milestone_project_tic-tac-toe.html
I couldn't figure out how to make ts not complain - will merge for now and revisit.

Reply:
thx, the type definitions were just off